src/toggl_api.cc

```diff
@@ -1017,6 +1017,11 @@ bool_t toggl_timeline_is_recording_enabled(
     return app(context)->IsTimelineRecordingEnabled();
 }
 
+char_t* toggl_time_of_day_format(
+    void* context) {
```

Comment:
The `*` should be to the left of the member name as everywhere, not to the right of the type name.
One issue is that our API generation will fail if the formatting is incorrect.

`make format` will fix this but it will probably change too many things so this better be fixed manually.